### ISSUE TITLE
Question: Range for TimeInput

### ISSUE THREAD

**Issue by pranaytripathi** [arfedulov/semantic-ui-calendar-react#37]
Does the TimeInput component has a range for time selection. For example, I want to enable time between 11:00 to 15:00. All values apart from that should be disabled.

**Comment by arfedulov** [comment]
No, it doesn't. I was going to add this feature. I'll try to find time for this on this weekend.

**Comment by arfedulov** [comment]
I didn't started yet. So it would be great if you take it.
